lib/ember-qunit/qunit-module.js

```diff
@@ -33,9 +33,15 @@ export function createModule(Constructor, name, description, callbacks) {
   var beforeEach = beforeEachCallback(callbacks || description);
   var afterEach  = afterEachCallback(callbacks || description);
 
-  var module = new Constructor(name, description, callbacks);
+  var module;
+
+  qunitModule(name, {
+    before() {
```

Comment:
This will also need to support user-defined `before`/`after` hooks, otherwise we potentially break existing tests.